Simplify the CSS Logical tests and prevent the result log from being rotated

Loirooriol opened the pull request:
Cc @mrego

Loirooriol committed:
Simplify the CSS Logical tests and prevent the result log from being rotated.

mrego commented on css/css-logical/logicalprops-block-size-vlr.html:
```diff
@@ -93,75 +72,34 @@
-  assert_true(compareWidthHeight("table3_cell", "ref_table3_cell"));
-}, "Check that block-size < min-block-size or min-block-size < block-size <= max-block-size or block-size > max-block-size in table vertical-lr");
+// Check that block-size < min-block-size or min-block-size < block-size <= max-block-size or block-size > max-block-size in table vertical-lr
+checkLayout(".tablecell");
```
If you want to call `checkLayout()` more than once, you need to pass `false` as 2nd argument.
And then call manually `done()` at the end of your test.

Otherwise you can change numbers and you'll see that things don't fail as expected.

mrego commented on css/css-logical/logicalprops-block-size-vlr.html:
```diff
@@ -93,75 +72,34 @@
-<div id="ref_div1"></div>
-<div id="ref_div2"></div>
-<div id="ref_div3"></div>
+<div id="tests">
```
I'd add `<div id="log"></div>` at the beginning of the test, so the results would appear there.
Then you could put the writing-mode directly on `#tests` and avoid repeating it on the rest of elements.

mrego reviewed (commented):
Thanks for doing these changes.
I have 2 comments but it's looking good.

Loirooriol committed:
Add test descriptions and fix issues

Loirooriol commented:
I added the descriptions in `<p>`, which were used for some tests, so instead I added classes and renamed some things. Didn't set `writing-mode` on the parent to avoid affecting the descriptions.

mrego commented on css/css-logical/logicalprops-block-size-vlr.html:
```diff
@@ -93,75 +72,52 @@
-<p id="ref_p1"></div>
-<p id="ref_p2"></div>
+<h3>Maximum and minimim block sizes in blocks with vertical-lr</h3>
+<div class="tests">
```
Where is this "tests" class defined?

Loirooriol replied:
`.tests` is used in logicalprops-block-size.html in order to define the width of the containing block. Other files don't assign styles to it but I still used it for consistency and to group similar tests. But I guess I can remove the class from the element.

mrego commented on css/css-logical/logicalprops-block-size-vlr.html:
```diff
@@ -93,75 +72,52 @@
+<div class="tests">
+  <p><code>block-size</code> &lt; <code>min-block-size</code></p>
+  <div class="table">
+    <div class="tablecell" id="table1_cell" data-expected-client-width="40" data-expected-client-height="100"></div>
```
Nit: "table-cell" sets again the vertical writing-mode, I don't think that's needed as it's already set in "table".

mrego commented on css/css-logical/logicalprops-block-size-vlr.html:
```diff
@@ -93,75 +72,52 @@
+checkLayout(".block", false);
+checkLayout(".override", false);
+checkLayout(".tablecell", false);
+done();
```
Nit: I believe you can change this for `checkLayout(".block,.override,.tablecell")`.
Or instead of having a `<script>` tag do it in `<body onload="checkLayout(".block,.override,.tablecell")">`.

Loirooriol replied:
I tried that, but then the log just shows `.block,.override,.tablecell` everywhere. If you separate them, then you see there are 3 `.block`, 2 `.override` and 3 `.tablecell`; that seemed better to me.

mrego reviewed (commented):
Thanks, a few more comments.

Loirooriol committed:
Remove unused class and unnecessary style

mrego reviewed (approved):
Thanks for doing the changes. LGTM!
